### DISCUSSION TITLE
186C7909-05C5-A6A7-625D-FF9E94C9F080

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#26] [General]
# Fifth Giant

Fifth Giant is a hypothetical icy giant planet that may have existed between the orbits of Saturn and Uranus. Therefore, at a time of unstable orbits of the giant planets, a possible encounter with Jupiter could have ejected the Fifth Giant out of the Solar System.

https://celestia.mobi/addon?item=186C7909-05C5-A6A7-625D-FF9E94C9F080&lang=en

**Comment by Dirgham2015** [comment]
Fifth giant is my favorite

**Comment by keizer12** [comment]
why is texture very different in game it looks like a exoplanet

**Reply by DaveBowman2001** [reply]
This add-on uses an old default exoplanet texture when it was made, it doesn't have a custom map of its own. Since 2023, that said texture has been replaced
